Source/Core/Core/HW/WiimoteReal/WiimoteReal.cpp

```diff
@@ -951,7 +925,10 @@ static void TryToConnectBalanceBoard(std::unique_ptr<Wiimote> wm)
 
 static void HandleWiimoteDisconnect(int index)
 {
-  g_wiimotes[index] = nullptr;
+  Core::RunAsCPUThread([index] {
+    g_wiimotes[index] = nullptr;
```

Comment:
I think this is not locked when it is called via `Update` ?

Reply:
Good call.